pkg/loki/modules.go

```diff
@@ -1133,14 +1137,14 @@ func (t *Loki) initRuleEvaluator() (services.Service, error) {
 func (t *Loki) initMemberlistKV() (services.Service, error) {
 	reg := prometheus.DefaultRegisterer
 
-	t.Cfg.MemberlistKV.MetricsNamespace = "loki"
+	t.Cfg.MemberlistKV.MetricsNamespace = t.Cfg.MetricsNamespace
```

Comment:
should we leave this one as is? since it is using the correct prefix already

Reply:
Good point.